src/components/EntityInfo.vue

```diff
@@ -269,7 +269,9 @@ export default class EntityInfo extends Mixins(AllowableActionsMixin, CommonMixi
     if (!this.isGoodStanding) {
       this.emitNotInGoodStanding(NigsMessage.CHANGE_COMPANY_INFO)
     } else {
-      const url = `${this.editUrl}${this.getIdentifier}/alteration`
+      let url = `${this.editUrl}${this.getIdentifier}`
+      // Append appropriate route based on entity type
+      url += (this.isFirm ? '/change' : '/alteration')
```

Comment:
Fairly certain Firms are/will be the exception. Majority of entity types will be alterations.

Reply:
I think so yea
We were explicit with Alterations, so i figured we could do the same with change filings. 
This could change in the future but this seems pretty straightforward for now :)

Reply:
Sounds good.